src/app/views/collections/mapper/collectionMapper.js

```diff
@@ -100,6 +100,15 @@ export default class collectionMapper {
                             uri: page.uri,
                             type: page.type,
                         };
+                        // if (this.props.isNewSignIn) {
+                        //     user.getUserEmail(updatedPage.lastEdit.email)
+                        //         .then(response => {
+                        //             return response;
+                        //         })
+                        //         .then(data => {
+                        //             updatedPage.lastEdit.email = data.email;
+                        //         });
+                        // }
```

Comment:
Thought about putting it here too with the idea of making the users/id requests a part of the mapping process however the way the application is structured I do not think its possible to have access to the environment variables i.e. isNewSignIn in this case.